### PR TITLE
Adding travis build file and yarn lock file

### PR DESCRIPTION
**Pull request by tocco934**

### CODE REVIEW

**Comment by yahehe** [comment]
Before you add this as a code change, does github have built in travis support? I ask mainly because the other repos I maintain have built in support for appveyor so there's no need for code changes to modify builds

**Comment by tocco934** [comment]
^ Talked in person, travis requires the travis.yml file to be present

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: node_js
```
Comment (croy) [inline .travis.yml]: line endings?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - node_modules
+notifications:
+  email: false
+node_js:
+- '6.10'
+before_script:
+- npm run lint
+script:
+  - if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then npm test; fi
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then git config user.email "tocco934+npm@gmail.com" && git config user.name "Dan Tocco"; fi
```
Comment (croy) [inline .travis.yml]: I presume this is temporary

Comment (tocco934) [inline .travis.yml]: Yup

Comment (yahehe) [inline .travis.yml]: But how temporary

Comment (yahehe) [inline .travis.yml]: Also nice use of +

Comment (tocco934) [inline .travis.yml]: Temporary as in, doesn't work so falling back to manual releases!

File: .travis.yml (hunk 1 above)
Comment (tocco934) [inline .travis.yml]: Fixed

**Comment by tocco934** [comment]
This is ready for review

File: readme.md [hunk 3]
```diff
@@ -91,10 +91,29 @@ You can run tests via `npm test`.
...
 
-## Credits
+## Making Changes
+
+### Contributing
+To contribute to this library fork the repo and create a pull-request back to the master branch.
+
+All changes require tests to be added or updated as part of the PR.
+
+Once Travis successfully builds your code, runs it through the linter, and runs all tests your PR will require approval before merging.
```
Comment (benjamingill) [inline readme.md]: nb: semantics: what does "builds your code" mean?

Comment (tocco934) [inline readme.md]: Good point, it means nothing haha

Comment (tocco934) [inline readme.md]: Updated

**Pull request merged**